src/averageAges.js

```diff
@@ -1,5 +1,23 @@
 'use strict';
 
+function getChildren(who, arr) {
+  return who.filter(
+    person => arr.some(one => one.name === person.mother)
+  );
+}
+
+function getAge({ died, born }) {
+  return died - born;
+}
+
+function getSex(arr, letter) {
```

Comment:
```suggestion
function getBySex(people, sex) { 
```
you don't get sex here, you filter people by their sex